setup.py

```diff
@@ -6,11 +6,78 @@
 import os, sys
 from os import chdir, getcwd
 from os.path import abspath, dirname, split, samefile
-import shlex
+import shlex, glob
 from subprocess import check_output
 
 import re
 
+# Distribution target: 'all', 'ndt', 'nd'
+DIST_TARGET = 'all'
```

Comment:
Maybe this could use a bit more of an expanded comment explaining the intended use, i.e. under what circumstance someone may want to change it.